sway/commands/exec_always.c

```diff
@@ -51,7 +51,35 @@ struct cmd_results *cmd_exec_always(int argc, char **argv) {
 	if ((pid = fork()) == 0) {
 		// Fork child process again
 		setsid();
+
 		if ((*child = fork()) == 0) {
+			// Acquire the current PATH
+			char *path = getenv("PATH");
+			const char *extra_path = ":" RUNDIR;
+			const size_t extra_size = sizeof(extra_path);
+
+			if (!path) {
+				size_t n = confstr(_CS_PATH, NULL, 0);
+				path = malloc(n + extra_size);
+				if (!path) {
+					return cmd_results_new(CMD_FAILURE, "exec_always", "Unable to allocate PATH");
+				}
+				confstr(_CS_PATH, path, n);
+
+			} else {
+				size_t n = strlen(path) +1;
+				char *tmp = malloc(n + extra_size);
+				strncpy(tmp, path, n);
```

Comment:
Copied to a buffer that is immediately discarded (and leaked). You need to add `path = tmp;`.

Reply:
Arg, yes, this was lost during rebase  ;_;
(I just checked, still had it in the `.deb` I built earlier :()